watch-package.js

```diff
@@ -8,16 +8,24 @@ var through = require('through2')
 var npm = process.platform === 'win32' ? 'npm.cmd' : 'npm';
 var nodemon = process.platform === 'win32' ? 'nodemon.cmd' : 'nodemon';
 
-module.exports = function watchPackage(pkgDir, exit) {
+var pkgDir = '';
+var stdin = null;
+
+module.exports = function watchPackage(_pkgDir, exit, taskName) {
```

Comment:
can taskName ever come in with only white-space?
if so, it should be trimmed as soon as possible